lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -253,10 +254,27 @@ def add_3ds_2_optional_fields(post, options)
 
       private
 
+      def isk_change_check(money)
+        if test?
+          isk_change_time = Time.new(2021, 8, 22, 8, 0, 0, '+00:00')
```

Comment:
Is there a "cost" of instantiating the same static date for every method call? If so, could these 2 values just be frozen and used instead?

Reply:
Possibly - I think that's a good idea.